requirements.txt

```diff
@@ -1,3 +1,4 @@
+algoliasearch>=2.0,<3.0
```

Comment:
We'll want to pin this to a specific version. I understand that they use semver so we're basically just creating more work for ourselves by pinning it rather than letting it get upgraded to the latest minor version automatically, but I still think that it makes more sense to pin it rather than keep it variable.

@wimo7083 What do you think? Should we specify a more specific version of Algolia or call it good at "between 2 and 3"? FWIW, the Algolia docs specify the way he has it https://www.algolia.com/doc/api-client/getting-started/update-the-client/python/?language=python#with-pip